public/app/plugins/datasource/cloudwatch/query-runner/CloudWatchLogsQueryRunner.ts

```diff
@@ -277,32 +322,32 @@ export class CloudWatchLogsQueryRunner extends CloudWatchRequest {
   ): Observable<DataFrame[]> {
     const range = options?.range || this.timeSrv.timeRange();
 
-    const requestParams = {
-      from: range.from.valueOf().toString(),
-      to: range.to.valueOf().toString(),
-      queries: queryParams.map((param: CloudWatchLogsRequest) => ({
-        // eslint-ignore-next-line
-        refId: (param as StartQueryRequest).refId || 'A',
+    const requestParams: DataQueryRequest<CloudWatchLogsQuery> = {
+      ...options,
+      range,
+      skipQueryCache: true,
```

Comment:
would we also need to skip query caches for metrics calls not just logs?

Reply:
no, we skip the query cache for logs because it's handled asynchronously. a typical flow for logs queries is:

1. make a request to start the query -> returns a query id
2. make a request to poll for the results -> returns the status of the query as 'running'
3. make a request to poll for the results -> returns the data as the query is now 'complete'

we need to skip the query cache otherwise step 2 is cached and we would just keep looping on that step since the cached query would also return the status of the query as 'running'